Dockerfile.s390x

```diff
@@ -0,0 +1,91 @@
+# Dockerfile for vLLM on s390x architecture with dependencies, PyTorch, torchvision, and Apache Arrow
+
+FROM ubuntu:22.04
+USER root
+
+
+RUN apt-get update -y && apt-get install -y \
+    git wget curl vim libnuma-dev libsndfile-dev libprotobuf-dev protobuf-compiler \
+    build-essential ffmpeg libsm6 libxext6 libgl1 python3 python3-pip cmake ninja-build \
+    cargo libjpeg-dev libpng-dev zlib1g-dev libavcodec-dev libavformat-dev libswscale-dev \
+    libtiff-dev libwebp-dev llvm-dev libclang-dev clang libssl-dev g++ \
+    python3-distutils python3-setuptools libbz2-dev liblz4-dev libzstd-dev \
+    libsnappy-dev rapidjson-dev libboost-dev liborc-dev pkg-config libopenblas-dev
+
+
+RUN python3 -m pip install --upgrade pip setuptools
+
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
+    . "$HOME/.cargo/env"
+
+
+WORKDIR /tmp/arrow
+RUN git clone https://github.com/apache/arrow.git && \
+    cd arrow/cpp && \
+    mkdir release && cd release && \
+    cmake -DCMAKE_BUILD_TYPE=Release \
+          -DCMAKE_INSTALL_PREFIX=/usr/local \
+          -DARROW_PYTHON=ON \
+          -DARROW_PARQUET=ON \
+          -DARROW_ORC=ON \
+          -DARROW_FILESYSTEM=ON \
+          -DARROW_WITH_LZ4=ON \
+          -DARROW_WITH_ZSTD=ON \
+          -DARROW_WITH_SNAPPY=ON \
+          -DARROW_JSON=ON \
+          -DARROW_CSV=ON \
+          -DPROTOBUF_PROTOC_EXECUTABLE=/usr/bin/protoc \
+          -DARROW_DEPENDENCY_SOURCE=BUNDLED \
+          .. && \
+    make -j$(nproc) && make install
+
+
+ENV CMAKE_PREFIX_PATH=/usr/local/lib/cmake:$CMAKE_PREFIX_PATH
+ENV LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+
+
+RUN pip install --no-cache-dir https://download.pytorch.org/whl/nightly/cpu/torch-2.6.0.dev20241201%2Bcpu-cp310-cp310-linux_s390x.whl
```

Comment:
any chance this could install torch from requirements instead of hard-coding dep here?